Web/starter_project_iconics/components/common/blogCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactElement } from 'react'
+import styles from '../../styles/blogCard.module.css'
+import { Blog } from '../../types/models'
+import { MdOutlineModeComment } from 'react-icons/md'
+
+import {
+  AiFillClockCircle,
+  AiFillCheckCircle,
+  AiFillWarning,
+} from 'react-icons/ai'
+
+interface Props {
+  card: Blog
+  feature: string
+}
+
+const BlogCard: React.FC<Props> = ({ card, feature }) => {
+  const handleReadMore = () => {
+    console.log('Read More button clicked')
+  }
+  const statusIcons: ReactElement[] = [
+    <AiFillCheckCircle key={1} size={20} className={styles.statIcon} />,
+    <AiFillWarning key={2} size={20} className={styles.statIcon} />,
+    <AiFillClockCircle key={3} size={20} className={styles.statIcon} />,
+  ]
+  const statusToNum: (stat: string) => number = (stat) => {
+    if (stat === 'approved') return 0
+    if (stat === 'denied') return 1
+    return 2
+  }
+  const formatNumber: (num: number) => string | number = (num: number) => {
```

Comment:
is there maybe a number formatting library you can use here ?
if possible that would be a nice idea

[nit] if you want to shorten this code you can use a similar method as above and define the letters "K", "M", "B" to a hashmap and index them by using some log math to get the index (just a suggestion for a coding challenge)

Reply:
I have found a way that is not so different but looks simpler and now it is changed